api/src/components/additionalInformation/__tests__/createAdditionalInformation.test.ts

```diff
@@ -0,0 +1,114 @@
+            'You can only add additional information to a draft publication'
+        );
+    });
+
+    test('User cannot add additional information to a DRAFT publication version they do not own', async () => {
+        const createAdditionalInformation = await testUtils.agent
+            .post('/publication-versions/publication-problem-draft-v1/additional-information')
+            .query({ apiKey: '123456789' })
+            .send({
+                title: 'Example title',
+                url: 'https://jisc.ac.uk'
+            });
+
+        expect(createAdditionalInformation.status).toEqual(403);
+        expect(createAdditionalInformation.body.message).toEqual(
+            'You cannot add additional information to a publication version you do not own'
+        );
+    });
+
+    test('User cannot create additional information without supplying a title', async () => {
+        const createAdditionalInformation = await testUtils.agent
+            .post('/publication-versions/publication-problem-draft-v1/additional-information')
+            .query({ apiKey: '000000005' })
+            .send({
+                url: 'https://jisc.ac.uk'
+            });
+
+        expect(createAdditionalInformation.status).toEqual(422);
+    });
+
+    test('User cannot create additional information without supplying a URL', async () => {
+        const createAdditionalInformation = await testUtils.agent
+            .post('/publication-versions/publication-problem-draft-v1/additional-information')
+            .query({ apiKey: '000000005' })
+            .send({
+                title: 'Example title'
+            });
+
+        expect(createAdditionalInformation.status).toEqual(422);
+    });
+
+    test('User cannot supply a description over 255 characters long', async () => {
+        const createAdditionalInformation = await testUtils.agent
+            .post('/publication-versions/publication-problem-draft-v1/additional-information')
+            .query({ apiKey: '000000005' })
+            .send({
+                title: 'Example title',
+                url: 'https://jisc.ac.uk',
+                description:
+                    'This is a description that is 256 characters long. It has to be this long in order to test the validation of the create external resource API endpoint. If this request gets rejected with a status code of 422, that means that the validation has rejected it.'
+            });
+
+        expect(createAdditionalInformation.status).toEqual(422);
+    });
+
+    test('User cannot supply an invalid URL', async () => {
+        const createAdditionalInformation = await testUtils.agent
+            .post('/publication-versions/publication-problem-draft-v1/additional-information')
+            .query({ apiKey: '000000005' })
+            .send({
+                title: 'Example title',
+                url: 'an ordinary string'
+            });
+
+        expect(createAdditionalInformation.status).toEqual(403);
```

Comment:
Would expect this to be 400 (or 422, whichever you decide). This isn't a permissions thing it's a validation thing.

Providing a url - correct request
Not providing a url or providing an invalid url - incorrect request

User has permission to create that additional information, they just requested it incorrectly.

Reply:
Should be fixed as per above comment.